packages/app/src/app/overmind/namespaces/editor/state.ts

```diff
@@ -120,6 +121,7 @@ export const state: State = {
   quickActionsOpen: false,
   previewWindowVisible: true,
   statusBar: true,
+  recoveredFiles: [],
```

Comment:
(needs some new imports, but this should be the type).

```suggestion
  recoveredFiles: Array<{ recoverData: RecoverData; module: Module }>,
```